rolecall/rolecall.py

```diff
@@ -333,7 +380,7 @@ def _get_object_by_name(self, otype, server, name, ignore_case=True):
         """ returns object if it exists, otherwise create the object """
         if object_type == "role":               # for roles
             role = discord.utils.get(server.roles, name=object_name)
-            try:                                # try in case role = None
+            try:                                #       try in case role = None
```

Comment:
ok?
please don't blindly commit ;)

Reply:
this is what fixed the spamming problem for me. lol jk

Reply:
fixed redundant indentation as of 37d82d4